### ISSUE TITLE
Does your training method apply intermediate supervision?

### ISSUE THREAD

**Issue by gavinwxy** [1adrianb/face-alignment-training#19]
Hi, I am a newcomer of face alignment work and trying to train your model on my own data set. I read about hourglass model and find it is trained by intermediate supervision in previous work. I read your torch 7 training code (I am actually not familiar with torch 7) and still can not find the answer. Could your please tell me if you compute loss for each hourglass stage and do backprop or just for the final stage. Thx to your work!
